README.md

```diff
@@ -43,20 +43,28 @@ Log into your Travis CI account and navigate to The National Archives organisati
 
 This repository is configured to allow for easy integration with Travis CI (Continuous Integration).
 
-#### 1.4.1 Obtaining dependencies via Composer
+#### 1.4.1 Grunt
 
-Having followed the steps above you will be able to install dependencies by typing ```composer install``` at the Terminal.
+Assuming that the [Grunt CLI](https://gruntjs.com/using-the-cli) has been installed follow the instructions on the [Grunt website](http://gruntjs.com/getting-started#working-with-an-existing-grunt-project).
+
+There are several Grunt tasks available to developers. View the Gruntfile.js for a listing of included tasks
+
+* ```grunt``` will run a predefined set of tasks and result in an ongoing 'watch' tasks (if enabled) which responds to any changes to JavaScript and SASS files by compiling, concatenating and minifying the output.
 
 #### 1.4.2 Obtaining dependencies via NPM
 
 Type ```npm install``` to obtain Node dependencies
 
+Type ```npm install grunt-contrib-sass --save-dev``` to compile Sass to CSS
+
+Type ```npm install grunt-contrib-cssmin --save-dev``` to minify CSS
```

Comment:
Hi @domingobishop - I don't think these two additional commands are necessary because these dependencies (`grunt-contrib-sass` and `grunt-contrib-cssmin`) are already listed in the `package.json` file (so they'd be pulled down from NPM by the first call to `npm install`).

Reply:
@gtvj They didn't install for me. So I had to do it manually. Don't know why. I'll add an 'if' to the readme.